Sources/Datadog/Core/Upload/DataUploader.swift

```diff
@@ -14,14 +14,6 @@ internal class UploadURLProvider {
     class QueryItemProvider {
         let value: () -> URLQueryItem
 
-        /// Creates `batch_time=...` query item adding current timestamp (in milliseconds) to the URL.
-        static func batchTime(using dateProvider: DateProvider) -> QueryItemProvider {
-            return QueryItemProvider {
-                let timestamp = dateProvider.currentDate().timeIntervalSince1970.toMilliseconds
-                return URLQueryItem(name: "batch_time", value: "\(timestamp)")
-            }
-        }
```

Comment:
we introduced those dynamic providers because of `batchTime`, now they can be static actually.
maybe here we can leave a note or something.?

Reply:
👌 yeap, added.